From f0f527ff3815caa091be24168824f74853c0c050 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 22 Mar 2022 10:40:08 -0400
Subject: [PATCH] local-display-factory: Fix type of signal connection id

Signal connection ids are 64-bit not 32-bit.

This commit fixes the type used.
---
 daemon/gdm-local-display-factory.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-local-display-factory.c b/daemon/gdm-local-display-factory.c
index 7de7b99c..4a66ad8f 100644
--- a/daemon/gdm-local-display-factory.c
+++ b/daemon/gdm-local-display-factory.c
@@ -78,7 +78,7 @@ struct _GdmLocalDisplayFactory
         gboolean         seat0_graphics_check_timed_out;
         guint            seat0_graphics_check_timeout_id;
 
-        guint            uevent_handler_id;
+        gulong           uevent_handler_id;
 
 #if defined(ENABLE_USER_DISPLAY_SERVER)
         unsigned int     active_vt;
-- 
2.35.1

